rllib/policy/tf_policy.py

```diff
@@ -253,11 +251,16 @@ def learn_on_batch(self, postprocessed_batch):
 
     @override(Policy)
     def get_weights(self):
-        return self._variables.get_flat()
+        variables = self.model.variables()
+        return [v.numpy() for v in variables]
```

Comment:
Wouldnt this line only work if eager is enabled? Should it check for that?

Reply:
Confirming, training a model without eager gives a `numpy is only enabled in eager mode` error

Reply:
My bad, I think this should be fixed now. Try it again?